App.js

```diff
@@ -6,38 +6,488 @@ import styles from './src/style';
-				<RouteSelection/>
-			</View>
-		);
-	}
+export default class BuildingMarker extends Component {
+  
+  constructor(props){
+          super(props);
+          this.state = {isLoading: true, scrollable: false , data:[]};
+     }
+  
+     componentDidMount(){
+         var paths = this.props.filenames.reduce(
+  ["10Church",
+     "amundson",
+     "universityAveRamp",
+     "eastRiverGarage",
+     "ford",
+     "admundson",
+     "akerman",
+     "diehl",
+     "boynton",
+     "dVC",
+     "civilE",
+     "childrenRehabClinic",
+     "hsec",
+     "keller",
+     "lind",
+     "mayoBuilding",
+     "mcb",
+     "moosTower",
+     "nilsHasselmo",
+     "nolteCtr",
+     "northrop",
+     "phillipsWangensteen",
+     "rapson",
+     "shepherdLab",
+     "tate",
+     "vincent",
+     "hospitalRamp",
+     "washingtonAveRamp",
+     "weaverDensford",
+     "akerman",
+     "coffman",
+     "folwell",
+     "graduate",
+     "hasselmo",
+     "jackson",
+     "jones",
+     "keller",
+     "kolthoff",
+     "lind",
+     "mCC",
+     "mcNamara",
+     "mechE",
+     "smith",
+     "nolteCtrGarage",
+     "10Church",
+     "folwell",
+     "morill",
+     "murphy",
+     "nolteCtr",
+     "northropGarage",
+     "tate",
+     "tNs",
+     "universityAveRamp",
+     "varClub",
+     "walter",
+     "washingtonAveRamp",
+     "williamson",
+     "yudof",
+     "johnston",
+     "mayoBldg"]);
+          var urls = paths.map(path => "https://sase-labs-2020.github.io/assets/informations/" + path.join("_") + ".json");
+          urls = urls.forEach(url =>
+               {return fetch(url,
+                         {
+                              method: "GET",
+                              headers: {
+                              'Accept': 'application/json',
+                              'Content-Type': 'application/json',
+                              },
+                         }
+                    ).then(response => response.json())
+                    .then((responseData) => {
+                         console.log(responseData);
+                              this.setState(
+                                   (prevState) => {
+                                        return {
+                                             data: prevState.data.concat(responseData),
+                                             isLoading: false,
+                                        };
+                                   }
+                              );
+                         });
+          });
+     }
+  
+  
+render() {
+  return (
+    <MapView
+      style={{ flex: 1 }}
+      showsUserLocation
+      initialRegion={{
+          latitude: 44.9727,
+          longitude: -93.2354,
+          latitudeDelta: 0.015,
+          longitudeDelta: 0.0121,
+      }}>
+     
+      <Marker
+          coordinate={{
+          latitude: 44.9727,
+          longitude: -93.2354}}
+          title={'Coffman Union'}
+          description={''}
```

Comment:
Just a reminder that you'll have to pull in the data from various sources. The coordinates come from `assets/directions`, while everything else comes from `assets/informations`. Also a reminder that we have to filter down the directions files because there only needs to be one marker for each building (even though there may be more than one path starting at a building).